extensions/spark/kyuubi-spark-authz/pom.xml

```diff
@@ -380,6 +380,19 @@
             <artifactId>${hudi.artifact}</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.scalatestplus</groupId>
+            <artifactId>mockito-4-11_${scala.binary.version}</artifactId>
+            <version>${scalatestplus.version}</version>
```

Comment:
```suggestion
```

Reply:
we prefer to manage all deps version/exclusions in root pom's `dependencyManagement` block, and simply reference the groupId/artifactId/scope in the sub-modules pom's `dependencies` scope